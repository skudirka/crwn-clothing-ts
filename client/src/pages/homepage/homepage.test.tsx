import {shallow, mount} from 'enzyme';
import React from 'react';

import HomePage from './homepage.component';

describe('HomePage page', () => {

    it('should render HomePage page', () => {
        expect(shallow(<HomePage />)).toMatchSnapshot();
    });
})