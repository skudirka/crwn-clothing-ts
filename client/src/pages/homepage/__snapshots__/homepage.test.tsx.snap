// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage page should render HomePage page 1`] = `ShallowWrapper {}`;
